deploy/operator.yaml

```diff
@@ -238,7 +238,7 @@ spec:
       serviceAccountName: kubevirt-ssp-operator
       containers:
         - name: kubevirt-ssp-operator
-          image: quay.io/fromani/kubevirt-ssp-operator-container:v1.0.4
+          image: quay.io/fromani/kubevirt-ssp-operator-container:v1.0.7
```

Comment:
would like to move away from storing images in personal repos.

Reply:
Agreed, Unfortunately we didn't make it in time, the ssp-operator is not yet under the kubevirt umbrella.

Reply:
IOW, that's the only u/s operator image available :\